java/daikon/dcomp/DCInstrument24.java

```diff
@@ -166,15 +167,19 @@ public class DCInstrument24 {
   // Variables used for calculating the state of the operand stack.
 
   /** Mapping from a label to its index in the method's codeList. */
+  @SuppressWarnings("nullness:initialization.static.field.uninitialized") // what's the lifecyle?
   protected static Map<Label, Integer> labelIndexMap;
 
   /** Mapping from a label to its operand stack. */
+  @SuppressWarnings("nullness:initialization.static.field.uninitialized") // what's the lifecyle?
   protected static Map<Label, OperandStack24> worklistHistory;
 
   /** State of operand stack prior to each byte code instruction. */
+  @SuppressWarnings("nullness:initialization.static.field.uninitialized") // what's the lifecyle?
   protected static OperandStack24[] stacks;
 
   /** The type of each local variable. */
+  @SuppressWarnings("nullness:initialization.static.field.uninitialized") // what's the lifecyle?
```

Comment:
All four of these are defined near the beginning of instrumentCodeList and modified and consumed by various methods called from instrumentCodeList.  This means they are redefined for each method instrumented.

Also, same question about verification as the @SuppressWarnings("nullness") for the entire class still exists.